lightning/src/ln/channelmanager.rs

```diff
@@ -1108,6 +1108,10 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 	///
 	/// Raises APIError::APIMisuseError when channel_value_satoshis > 2**24 or push_msat is
 	/// greater than channel_value_satoshis * 1k or channel_value_satoshis is < 1000.
+	///
+	/// Note that we do not check if you are currently connected to the given peer. If no
+	/// connection is available, the outbound `open_channel` message may fail to send, resulting in
+	/// the channel eventually being silently forgotten.
```

Comment:
What do you mean by "being silently forgotten", can't remember this case is covered by `peer_disconnected` ?

Reply:
It is, it will be silently forgotten when we disconnect or restart.